orchagent/vnetorch.cpp

```diff
@@ -15,21 +15,23 @@
 #include "directory.h"
 #include "swssnet.h"
 #include "intfsorch.h"
+#include "neighorch.h"
 
 extern sai_virtual_router_api_t* sai_virtual_router_api;
 extern sai_route_api_t* sai_route_api;
 extern sai_object_id_t gSwitchId;
 extern Directory<Orch*> gDirectory;
 extern PortsOrch *gPortsOrch;
 extern IntfsOrch *gIntfsOrch;
+extern NeighOrch *gNeighOrch;
 
 /*
  * VRF Modeling and VNetVrf class definitions
  */
 std::vector<VR_TYPE> vr_cntxt;
 
-VNetVrfObject::VNetVrfObject(const std::string& vnet, string& tunnel, set<string>& peer,
-                             vector<sai_attribute_t>& attrs) : VNetObject(tunnel, peer)
+VNetVrfObject::VNetVrfObject(const std::string& vnet, string& tunnel, set<string>& peer, uint32_t vni,
```

Comment:
Maybe it is worth considering grouping all these parameters into a single struct VnetInfo?

Reply:
Addressed